CHANGES.md

```diff
@@ -1,3 +1,7 @@
+# 2.9.1
+
+* [+] Integration with AWS root account is not available anymore because of security and recommendation to avoid using this way.
```

Comment:
* Is it a new feature? For me, it looks more like removing the non-supported feature.
* Does Amazon no longer provides authentication using root account credentials? If so, is it only for Plesk or for others as well?

Reply:
Amazon supports, but we removing such way of auth for security reasons

Reply:
Amazon guys does not recommends such way of auth for regular tasks: https://docs.aws.amazon.com/signin/latest/userguide/introduction-to-root-user-sign-in-tutorial.html

Reply:
[*] [For the sake of security](https://docs.aws.amazon.com/signin/latest/userguide/introduction-to-root-user-sign-in-tutorial.html), we removed the ability to sign in to AWS using the account's root user credentials.